rskj-core/src/test/java/co/rsk/peg/BridgeSupportTest.java

```diff
@@ -323,6 +323,40 @@ public void getLockingCap() {
         verify(provider, times(1)).setLockingCap(constants.getInitialLockingCap());
     }
 
+    @Test
+    public void getActivePowpegRedeemScriptBeforeFork() {
+        ActivationConfig.ForBlock activations = mock(ActivationConfig.ForBlock.class);
+        when(activations.isActive(ConsensusRule.RSKIP293)).thenReturn(false);
+
+        Federation fed = bridgeConstants.getGenesisFederation();
+        FederationSupport federationSupport = mock(FederationSupport.class);
```

Comment:
What's the use of this `federationSupport` object?

Reply:
I just realized there is no need to create a Federation by my own 🤦‍♂️  if I use the `BridgeSupportBuilder` and pass to it a `BridgedConstants` instance .